### ISSUE TITLE
Page rendered incorrectly on Jimmy 1.2 / 2.0

### ISSUE THREAD

**Issue by ghost** [jfoucher/Jimmy#11]
The page below renders without the year 2018, and misplaces the link for "Tower 57" under 2019 instead. Visit gemini://cugi.ie/media/ to see, but it's under development and changing a little over time.

Could be an issue with consecutive headings without content (## 2019 -> ## 2018) ?

<img width="553" alt="Screenshot 2022-02-24 at 10 59 18 p m" src="https://user-images.githubusercontent.com/1757768/155621788-7c779a46-4af3-48a3-9d16-a610e4e7b937.png">

```
=> / Home
=> /art/ Art
=> /podcast/ Podcast
=> /hiscores/ Hi-scores
=> /newsletter/ Newsletter
=> /history/ History

# Media (UNDER CONSTRUCTION)
## 2022 
Event not held due to potential changes in pandemic restrictions. See you in 2023.

## 2021
Held online due to the pandemic.
=> /0.jpg Group Photo 1
=> /1.jpg Group Photo 2
=> /2.jpg Group Photo 3

## 2020
=> /crg.png Group 1
=> /2020-1.jpg Group 2
=> /2020-2.jpg Group 3

## 2019


## 2018
=> /bgo.jpg Amiga port of Tower 57 by Daniel 'Daytona675x' Müßener by Banjo Guy Ollie


## 2017
=> /aeon.png Group photo by Trevor Dickinson of A-EON Technology ltd

=> /LiamD-photo.png

## 2016
(missing 2016 photos)



=> mailto:event@cugi.ie Contact
=> https://retrocomputing.stackexchange.com/questions/tagged/amiga/ Tech Support
=> /policies/ Policies
=> /media/ Media
=> gemini://jeacle.ie/amiga/amigemini/ AmiGemini - Amiga Gemini Client
=> ircs://libera.chat:6697/#amiga-ie IRC

```

**Comment by jfoucher** [comment]
Yes it’s definitely an issue with title parsing. It will be solved with #10

**Comment by jfoucher** [comment]
Should be fixed in [v0.2.1](https://github.com/jfoucher/Jimmy/releases/tag/v0.2.1)
Can you check and let me know?
Thanks

**Comment by jfoucher** [comment]
Closing this for now, do not hesitate to reopen @tenleftfingers if you happen to notice this behaviour again.
